objects/obj_prompt/Step_0.gml

```diff
@@ -7,19 +7,32 @@ y += shift;
 switch fadeMe {
 	// fade in
 	case FADE_IN: {
-		if (image_alpha < 1)
+		if (image_alpha < 1) {
 			image_alpha += fadeSpeed;
-		if (image_alpha >= 1)
+		}
+		if (image_alpha >= 1) {
 			fadeMe = FADE_VISIBLE;
+		}
 		break;
 	}
 	case FADE_OUT: {
-		if (image_alpha > 0)
+		if (image_alpha > 0) {
 			image_alpha -= fadeSpeed;
+		}
 		if (image_alpha <= 0) {
 			fadeMe = FADE_DONE;
-			alarm[0] = 2 // queue destroy
+			alarm[0] = 2; // queue destroy
 		}
 		break;
 	}
+	
+	case FADE_VISIBLE: {
+		 // The object is now fully visible, do any continuous behavior here
```

Comment:
Have some weird formatting in here

Reply:
Also, these cases don't make any sense. Why is there nothing in them?

Reply:
Does it need to exist in order for what's in Alarm 0 to take affect?